File: Desen_Sis&Back-End/Type-3/JoseVitorSAmorim_1.js
const prompt = require("prompt-sync")();
let nome = prompt("Nome: ");
let nota = prompt("Nota: ");
let presenca = prompt("Presença(%): ");
if (presenca >= 50 && nota >= 80) {
    console.log(nome + " foi aprovado!");
}
else if (presenca >= 75 && nota >= 60) {
    console.log(nome + " foi aprovado!");
}
else {    
    console.log(nome + " foi reprovado!");
}

File: Desen_Sis&Back-End/Type-3/JoseVitorSAmorim_1.py
nome = str(input("Nome: "))
nota = float(input("Nota: "))
presenca = int(input("Presença(%): "))
if presenca >= 50 and nota >=80:
    print(f"Aluno {nome} aprovado")
elif presenca >=75 and nota >=60:
    print(f"Aluno {nome} aprovado")
else:
    print(f"Aluno {nome} reprovado")